examples/bzlmod/pip_repository_annotations_test.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import platform
+import subprocess
+import sys
+import unittest
+from pathlib import Path
+
+from python.runfiles import runfiles
+
+
+class PipRepositoryAnnotationsTest(unittest.TestCase):
+    maxDiff = None
+
+    def wheel_pkg_dir(self) -> str:
+        env = os.environ.get("WHEEL_PKG_DIR")
+        self.assertIsNotNone(env)
+        return env
+
+    def test_build_content_and_data(self):
+        r = runfiles.Create()
+        rpath = r.Rlocation(
+                "rules_python~override~pip~{}/generated_file.txt".format(
```

Comment:
You shouldn't need to put `~override~` in the runfile paths -- did just the repository name not work? It should be the repo name whose files you want to access

Reply:
I tried.  It does not work